test/apps/iodemo/io_demo.cc

```diff
@@ -786,7 +781,10 @@ class DemoClient : public P2pDemoCommon {
             ++(*_io_counter);
             ++(*_server_io_counter);
             if (_validate && (status == UCS_OK)) {
-                validate(*_iov, _sn);
+                iomsg_t *io_msg = reinterpret_cast<iomsg_t*>(_buffer);
+                unsigned seed   = io_msg->sn;
```

Comment:
can we remove "seed" and "io_msg" local vars

Reply:
need to cast buffer to `iomsg_t *` to validate iomsg, `seed` can be removed.